src/Sylius/Bundle/ApiBundle/Doctrine/QueryCollectionExtension/RestrictingFilterEagerLoadingExtension.php

```diff
@@ -13,32 +13,41 @@
 
 namespace Sylius\Bundle\ApiBundle\Doctrine\QueryCollectionExtension;
 
-use ApiPlatform\Core\Bridge\Doctrine\Orm\Extension\ContextAwareQueryCollectionExtensionInterface;
-use ApiPlatform\Core\Bridge\Doctrine\Orm\Util\QueryNameGeneratorInterface;
+use ApiPlatform\Doctrine\Orm\Extension\QueryCollectionExtensionInterface;
+use ApiPlatform\Doctrine\Orm\Util\QueryNameGeneratorInterface;
+use ApiPlatform\Metadata\Operation;
 use Doctrine\ORM\QueryBuilder;
 
 /**
  * @experimental
  * This class decorates api_platform.doctrine.orm.query_extension.filter_eager_loading.
  * It is a workaround for https://github.com/api-platform/core/issues/2253.
```

Comment:
Just wondering if we should check if its still valid after bump 🤔

Reply:
Good catch, but I think we should face this when converting a resource using this service